### PR TITLE
Soft policies - design document

### PR DESCRIPTION
**Pull request by shlomitk1**
Design for https://github.com/fybrik/fybrik/issues/1131

### CODE REVIEW

File: pkg/adminconfig/soft_policies.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+### Weights
+
+- A default value is 1.0 
+- All weights in a decision are normalized, i.e. weights 1, 0.5 and 0.5 result in weights 0.5, 0.25 and 0.25 respectively.
+
+### Conflict resolution
+
+In case of more than one optimization strategy, the first one takes precedence, and the others are ignored.
```
Comment (simanadler) [inline pkg/adminconfig/soft_policies.md:105]: In the future we will add priorities to the policies, which will help address this.  Right?

File: pkg/adminconfig/soft_policies.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    "object": "storageaccount",
+    "instance": "account-theshire"
+}
+
+{
+    "attribute": "bandwidth",
+    "description": "bandwidth between neverland and theshire",
+    "value": "5",
+    "type": "numeric",
+    "units": "Megabit/sec",
```
Comment (roytman) [inline pkg/adminconfig/soft_policies.md:54]: A tiny comment:
the ISO bandwidth abbreviations are:
Mbp/s, Gbp/s, Tbp/s for megabits, gigabits, terabits per second
and
MB/s, GB/s, TB/s for megabytes, gigabytes, and terabytes per sec.

Comment (shlomitk1) [reply pkg/adminconfig/soft_policies.md]: Thanks, changed.

File: pkg/adminconfig/soft_policies.md (hunk 1 above)
Comment (shlomitk1) [reply pkg/adminconfig/soft_policies.md]: Right, added a comment about this.

**Comment by shlomitk1** [comment]
Closing this PR, it is a part of https://github.com/fybrik/fybrik/pull/1323

**Pull request closed without merging**